manual/C/ch_Finance-Quote.xml

```diff
@@ -240,6 +251,199 @@ SYNOPSIS
     </procedure>
   </sect1>
 
+  <sect1 id="finance-quote-helper">
+    <title>Helper Scripts for Update and Diagnostics</title>
+
+    <abstract>
+      <para>&app; provides several utilities that support you in your daily work with &app-fq;.
+        So there is a script with which you can check your symbols for the securities. Other
+        scripts perform the initial installation or keep &app-fq; up to date.
+        &app-fq; software module. These programs have names that begin with gnc-fq-*, and are
+        themselves written in the &app-perl; programming language. 
+      </para>
+    </abstract>
+
+    <procedure>
+      <title>Running &app-perl; Scripts Under &win;</title>
+
+      <step>
+        <simpara>Open a <application>CMD</application> or <application>Powershell</application>
+          window: Click <guimenu>Start</guimenu> and type either
+          <userinput>cmd</userinput> or <userinput>powershell</userinput>
+          then select the resulting menu item. You don't need to run as administrator.
+        </simpara>
+      </step>
+
+      <step>
+        <simpara>Change to the drive containing the &app; program files —usually C drive—
+          if that is not already the current drive.
+        </simpara>
+      </step>
+
+      <step>
+        <simpara>Since, as previously recommended, the GnuCash directory was entered in the <envar>PATH</envar>
+          envinronment variable, prefix any gnc-fq-xxx &app-perl; script command with
+          <quote>Perl</quote> and a space, e.g. <userinput>perl gnc-fq-check</userinput>.
+          Otherweise you have to execute the command e.g. <userinput>perl c:\Program Files
+          <replaceable> (x86)</replaceable>\gnucash\bin\gnc-fq-check</userinput>.
```

Comment:
I don't think using the `<replaceable>` tag to mark ` (x86)` is helpful. Yes, it's true that some user might build GnuCash for themselves as 64-bit and install it in `C:\Progam Files\`, but that's unlikely. However the installer allows the user to install to any arbitrary path so it would make sense to mark the whole thing as replaceable:
`<userinput><replaceable>C:\Program Files (x86)\gnucash</replaceable>\bin\gnc-fq-check</userinput>`.